io/driver/module_unload_load.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
Would you please start with the license heading? You can inspire from other tests ins `avocado-misc-test` repo.